trunk/projects/h5fs_test/regen_test_data

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
I'm not sure if this is helpful with your defaulted command line arguments above. Also the bash scripts are not as useful as helper utilities because we should be able to test multi-platform. If we do need helper utilities, it is probably best to write them in python.